invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -18,14 +18,14 @@
 
 {%- block page_body %}
 <div class="banners">
-  {% if permissions is defined and permissions.can_edit %}
+  {% if permissions is defined and permissions.can_edit and not is_preview %}
   <div class="ui warning flashed top-attached manage message">
     <div class="ui container">
       <div id="recordManagement" data-record='{{ record | tojson }}' data-permissions='{{ permissions | tojson }}'></div>
     </div>
   </div>
   {% endif %}
-  {% if not record.versions.is_latest %}
+  {% if not record.versions.is_latest and not is_preview %}
```

Comment:
Minor: I think we should still show the "newer" version message on the preview, so that they'll see how it looks like